win32/devdoc/threadpool_win32.md

```diff
@@ -265,45 +265,45 @@ MOCKABLE_FUNCTION(, THREADPOOL_WORK_ITEM_HANDLE, threadpool_create_work_item, TH
+MOCKABLE_FUNCTION(, int, threadpool_schedule_work_item, THANDLE(THREADPOOL), threadpool, THANDLE(THREADPOOL_WORK_ITEM), threadpool_work_item);
 ```
 
 `threadpool_schedule_work_item` schedules a work item to be executed by the threadpool.
 
 **SRS_THREADPOOL_WIN32_05_011: [** If `threadpool` is NULL, `threadpool_schedule_work_item` shall fail and return a non-zero value. **]**
 
-**SRS_THREADPOOL_WIN32_05_012: [** If `work_item_context` is NULL, `threadpool_schedule_work_item` shall fail and return a non-zero value. **]**
+**SRS_THREADPOOL_WIN32_05_012: [** If `threadpool_work_item` is NULL, `threadpool_schedule_work_item` shall fail and return a non-zero value. **]**
 
 **SRS_THREADPOOL_WIN32_05_013: [** `threadpool_schedule_work_item` shall call `SubmitThreadpoolWork` to submit the work item for execution. **]**
 
-### threadpool_destroy_work_item
+### threadpool_dispose_work_item
 
 ```c
-MOCKABLE_FUNCTION(, void, threadpool_destroy_work_item, THANDLE(THREADPOOL), threadpool, THREADPOOL_WORK_ITEM_HANDLE, work_item_context);
+static void threadpool_dispose_work_item(THANDLE(THREADPOOL_WORK_ITEM) threadpool_work_item);
 ```
 
-`threadpool_destroy_work_item` closes the `ptp_work` member variable in `work_item_context`and then frees the `work_item_context`
+`threadpool_dispose_work_item` closes the `ptp_work` member variable in `threadpool_work_item`and then frees the `threadpool_work_item`
 
-**SRS_THREADPOOL_WIN32_05_014: [ ** If `threadpool` is `NULL`, `threadpool_destroy_work_item` shall fail and return a non-zero value. **]**
+**SRS_THREADPOOL_WIN32_05_014: [ ** [Deprecated] If `threadpool` is `NULL`, `threadpool_dispose_work_item` shall fail and return a non-zero value. **]**
 
-**SRS_THREADPOOL_WIN32_05_015: [** If `work_item_context` is `NULL`, `threadpool_destroy_work_item` shall fail and not do anything before returning. **]**
+**SRS_THREADPOOL_WIN32_05_015: [** [Deprecated] If `threadpool_work_item` is `NULL`, `threadpool_dispose_work_item` shall fail and not do anything before returning. **]**
 
-**SRS_THREADPOOL_WIN32_05_016: [** `threadpool_destroy_work_item` shall call `WaitForThreadpoolWorkCallbacks` to wait on all outstanding tasks being scheduled on this `ptp_work`. **]**
+**SRS_THREADPOOL_WIN32_05_016: [** `threadpool_dispose_work_item` shall call `WaitForThreadpoolWorkCallbacks` to wait on all outstanding tasks being scheduled on this `ptp_work`. **]**
 
-**SRS_THREADPOOL_WIN32_05_017: [** `threadpool_destroy_work_item` shall call `CloseThreadpoolWork` to close `ptp_work`. **]**
+**SRS_THREADPOOL_WIN32_05_017: [** `threadpool_dispose_work_item` shall call `CloseThreadpoolWork` to close `ptp_work`. **]**
 
-**SRS_THREADPOOL_WIN32_05_018: [** `threadpool_destroy_work_item` shall free the `work_item_context`. **]**
\ No newline at end of file
+**SRS_THREADPOOL_WIN32_05_018: [** [Deprecated] `threadpool_dispose_work_item` shall free the `threadpool_work_item`. **]**
```

Comment:
>** [](http://example.com/codeflow?start=121&length=2)

I've reached the maximum amount of contribution for this review, so I will gracefully sign off with the hope that progress will be made.